sdk/python/feast/inference.py

```diff
@@ -199,10 +199,10 @@ def _infer_features_and_entities(
         fv.batch_source.timestamp_field,
         fv.batch_source.created_timestamp_column,
     }
-    for column in columns_to_exclude:
-        if column in fv.batch_source.field_mapping:
-            columns_to_exclude.remove(column)
-            columns_to_exclude.add(fv.batch_source.field_mapping[column])
+    for k, v in fv.batch_source.field_mapping.items():
```

Comment:
nit: more descriptive variable names

maybe something like `original_column` and `mapped_column`?